scripts/apps/search/directives/SearchFilters.ts

```diff
@@ -38,6 +38,12 @@ class LinkFunction {
         };
         this.scope.dateFilters = getDateFilters()
             .filter((dateFilter) => metadata.search_config?.[dateFilter.fieldname] == null);
+        this.defaultFilterLabels = {
+            categories: gettext('CATEGORY'),
```

Comment:
Can you pass `Category`(and other labels) sentence-cased to `gettext` and uppercase using CSS? By uppercasing using CSS we can reduce the number duplicate of translations in transifex.